HW15/src/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,83 @@
+import java.io.File;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Date;
+
+public class FileLoggerConfiguration {
+
+    private static double maxFileSize = 0.3;
```

Comment:
Why is the file size double? It may be better to keep the file size in bytes (not byte java type, but bytes like raw size, e.g. 1024 - 1kb or 3221225472 - 3 GB).